packages/fxa-shared/subscriptions/stripe.ts

```diff
@@ -0,0 +1,136 @@
+import Stripe from 'stripe';
```

Comment:
Could use some tests for these methods. Looks like they're *used* in tests, but aren't themselves tested.